composer.json

```diff
@@ -21,12 +21,14 @@
     },
     "require": {
         "php": "^7.1",
-        "http-interop/http-server-middleware": "^1.0.1",
         "psr/container": "^1.0",
-        "psr/http-message": "^1.0.1"
+        "psr/http-message": "^1.0.1",
+        "psr/http-server-handler": "^1.0",
+        "psr/http-server-middleware": "^1.0",
+        "zendframework/zend-diactoros": "^1.7"
```

Comment:
Why new dependency to `zendframework/zend-diactoros` here?

Reply:
https://github.com/zendframework/zend-expressive-authentication/blob/release-1.0.0/src/ResponsePrototypeTrait.php#L14

Reply:
The whole point is not to have it there !

https://github.com/zendframework/zend-expressive-authentication/blob/release-1.0.0/src/ResponsePrototypeTrait.php#L26-L33

Reply:
https://github.com/zendframework/zend-expressive-authentication/blob/release-1.0.0/src/ResponsePrototypeTrait.php#L37